tests/migrations/test_operations.py

```diff
@@ -199,6 +199,42 @@ def test_create_model(self):
         definition = operation.deconstruct()
         self.assertNotIn('managers', definition[2])
 
+    def test_create_model_with_duplicate_field_name(self):
+        with self.assertRaises(ValueError):
```

Comment:
Please use `self.assertRaisesMessage` to verify this is the expected `ValueError` .